crates/router/src/types.rs

```diff
@@ -420,7 +420,7 @@ impl PayoutIndividualDetailsExt for api_models::payouts::PayoutIndividualDetails
 #[derive(Clone, Debug, Default)]
 pub struct PayoutsResponseData {
     pub status: Option<storage_enums::PayoutStatus>,
-    pub connector_payout_id: String,
+    pub connector_payout_id: Option<String>,
```

Comment:
TODO for later: make it non optional, and add new struct for connector integration if this field is not consumed in other flows